packages/micro-analytics-cli/src/handler.js

```diff
@@ -24,6 +24,13 @@ function realtimeHandler(req, res) {
   }
 }
 
+async function readMeta(req) {
+  if (await text(req)) {
+    return (await json(req)).meta;
```

Comment:
The issue is that it throws if the value is empty. We can wrap it in a try catch instead, but then it would not crash on bad data which would be confusing.